src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -182,23 +175,12 @@ def _extract_message(
 
         self.__message_count[use_case_id] += 1
 
-        self.__message_bytes_sum[use_case_id] += len(msg.payload.value)
-        self.__message_bytes_max[use_case_id] = max(
-            len(msg.payload.value), self.__message_bytes_max[use_case_id]
-        )
-
-        self.__message_tags_len_sum[use_case_id] += len(parsed_payload.get("tags", {}))
-        self.__message_tags_len_max[use_case_id] = max(
-            len(parsed_payload.get("tags", {})), self.__message_tags_len_max[use_case_id]
+        self.__message_bytes[use_case_id] += len(msg.payload.value)
+        self.__message_tags_len[use_case_id] += len(parsed_payload.get("tags", {}))
+        self.__message_value_len[use_case_id] += (
+            len(parsed_payload["value"]) if isinstance(parsed_payload["value"], Iterable) else 1
```

Comment:
Thanks for this. Indeed, we want to still count the presence of other metric types (other than just `set` and `distribution`). One point to consider though, might be that do we want to use `1` universally as so? For counters, this makes sense. I wonder if it does for gauges as well?

Reply:
Isn't the `value` field for gauges a `TypedDict`? Wouldn't that be an `Iterable`?

Reply:
That's right, yep